runtime/src/start.rs

```diff
@@ -48,7 +48,14 @@ impl Args {
     }
 }
 
-pub async fn start(loader: impl Loader + Send + 'static, runner: impl Runner + Send + 'static) {
+pub async fn start(
+    loader: impl Loader + Send + 'static,
+    runner: impl Runner + Send + 'static,
+    #[cfg_attr(not(feature = "setup-telemetry"), allow(unused_variables))]
+    project_name: &'static str,
+    #[cfg_attr(not(feature = "setup-telemetry"), allow(unused_variables))]
+    project_version: &'static str,
```

Comment:
nit: Name them `crate_name` and `package_version` to match cargo's names

Reply:
We can also make this more verbose: (a few lines below)
```rs
println!("{} starting: {} {}", crate::VERSION_STRING, crate_name, package_version);
```

Reply:
Both good suggestions, applied